src/AppInstallerCLITests/WorkFlow.cpp

```diff
@@ -305,25 +307,7 @@ namespace
         }
 
         // For clone
-        TestContext(std::ostream& out, std::istream& in, std::shared_ptr<std::vector<WorkflowTaskOverride>> overrides) :
-            m_out(out), m_in(in), m_overrides(overrides), m_isClone(true), Context(out, in)
-        {
-            m_shouldExecuteWorkflowTask = [this](const Workflow::WorkflowTask& task)
-            {
-                auto itr = std::find_if(m_overrides->begin(), m_overrides->end(), [&](const WorkflowTaskOverride& wto) { return wto.Target == task; });
-
-                if (itr == m_overrides->end())
-                {
-                    return true;
-                }
-                else
-                {
-                    itr->Used = true;
-                    itr->Override(*this);
-                    return false;
-                }
-            };
-        }
+        TestContext(std::ostream& out, std::istream& in, std::shared_ptr<std::vector<WorkflowTaskOverride>> overrides) : TestContext(out, in, true, overrides) {}

```

Comment:
>TestContext(std::ostream& out, std::istream& in, std::shared_ptr<std::vector<WorkflowTaskOverride>> overrides) : TestContext(out, in, true, overrides) {} [](http://example.com/codeflow?start=8&length=153)

nit: this could be removed then?